src/lib/util/Config.ts

```diff
@@ -13,6 +13,8 @@ export type EngineConfigContent = {
 	name: string;
 	disabled?: boolean;
 	targetPatterns: string[];
+	useDefaultConfig?: boolean; //TODO: this parameter doesn't make sense for PMD. Make it optional.
```

Comment:
Since we weren't supporting the local/global scope yet, I added the useDefaultConfig property so that a user could keep a value in overriddenConfigPath without deleting it and yet choose to not use it. Kind of like commenting it for a run and not commenting it for another. But that could be just me.